index.html

```diff
@@ -1037,109 +1037,64 @@
-   service itself.
+The service may also use innocent people's data in order to distinguish them from these
+bad actors.
 
-   <aside class="example" id="example-user-service-safety-tradeoff">
+<strong>However:</strong>
 
-   * A service might collect IP address data from its visitors in order to block any that
-     are attacking the service. While a successful attack might also harm users, for
-     example if it prevents them from using the service or reveals their private data to
-     the attacker, the service might still be primarily motivated by preventing damage to
-     itself.
+1. These principles need to be widely-agreed, not just details within a unilateral
+   terms-of-service. For example, in most [=contexts=], it's [=inappropriate=] for a
+   person to misrepresent themself in order to trick other people. On the other hand,
+   there's generally no principle against transparently adopting a pseudonym, even if a
+   site's terms of service prohibit it.
 
-   </aside>
+1. The service must take technical and legal measures to ensure the extra data it's
+   collecting to protect people cannot be used for other purposes, like to grow the
+   service.
 
-<div class="practice">
+</div>
 
-<p><span class="practicelab" id="principle-safety-with-privacy">As much as possible,
-services should provide both safety and privacy.</span></p>
+<aside class="example" id="example-sockpuppets" title="Sockpuppets">
 
-This may require the development and adoption of new technologies. For example, wider
-adoption of [[?WebAuthn]] could reduce account hijacking enough that services no longer
-need to fingerprint devices to prevent it.
+A person might want to sign up many accounts ("sockpuppets") on a service in order to
+trick other people into thinking a belief has more support than it really has. This
+violates the other people's rights to be free from manipulation, and so it's
+[=appropriate=] to collect enough extra data to identify these sockpuppets.
 
-</div>
+Similar measures are appropriate to resist
+[astroturfing](https://en.wikipedia.org/wiki/Astroturfing).
 
-<div class="practice">
+</aside>
 
-<p><span class="practicelab" id="principle-violation-sacrifices-privacy">When a
-person violates the [=principles=] of a [=context=] they're interacting with, the service
-operating that context may use their data in ways that would otherwise be
-[=inappropriate=] solely in order to protect other users of that service.</span></p>
+<aside class="example" id="example-children" title="Children's Services">
 
-Some extra use of personal data might also be [=appropriate=] in order to protect the
-service itself, but less than is appropriate to protect other people.
+Children using a service (or their guardians) may want to ensure their interaction with
+that service is only visible to other children. This can justify collecting ages from the
+service's users in order to exclude adults, but not to then use those ages for other
+purposes.
```

Comment:
Yes. Maybe we should be careful not to trivialize age verification. Note that voluntary determination of age is ineffective. Here are some things that could work:
- face recognition, with regular/spontaneous re-verification
- Robust (e.g. [IAL2](https://pages.nist.gov/800-63-3/sp800-63a.html)) verification of an adult guardian, who then attests to the age of their child.

Reply:
I don't believe the proposed text trivializes age verification or that specific technical details are required here. I also don't think our privacy principles document should recommend regular biometric collection and face recognition of children.

Reply:
We should be intentional about this: if we endorse collecting ages at all, that implicitly gives permission to do it in a way that anti-abuse folks think is effective. If we want to exclude using face recognition on children, we have to say that ... and then we probably shouldn't imply that sites can only use easily-evadable ways of checking ages.

We can always dodge the problem in this document by deleting the example, but it's still going to come up in the real world.

Reply:
https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/105.html#example-children now includes both the pro and con side of collecting ages.